clang-format_test.cpp

```diff
@@ -0,0 +1,112 @@
+/***************************************************************************
+ *
+ * test file for clang-format.
+ *
+ * created 22 Nov 2022
+ *
+ * This file is provided to enable evaluation of clang-format settings.
+ * From a format point of view it is deliberately rubbish!
+ * Please add any formatting situation of interest
+*
+   *
+ * DO NOT FORMAT BEFORE MERGE !!
+ *
+          ***************************************************************************/
```

Comment:
Header needs to be reformatted

Reply:
This is a file **deliberately** badly formatted.
The whole point is to see what a formatter does to it.